Python/UpeachyaKansakar/tasks/task1/show_result.py

```diff
@@ -0,0 +1,46 @@
+import os
+import csv
+import argparse
+import operator
+import datetime as dt
+
+def calculatePercentage (score, total):
+    if score and total :
+        return (float(score)/int(total) * 100)
+    else:
+        return 0
+
+def displayRecords(fileName):
```

Comment:
For naming conventions follow https://www.coding-guidelines.lftechnology.com/docs/python/files

Reply:
:ok: